workflows/raredisease.nf

```diff
@@ -131,6 +133,14 @@ workflow RAREDISEASE {
     // STEP 0: PREPARE GENOME REFERENCES AND INDICES.
     PREPARE_GENOME ( params.fasta )
 
+    if (params.gnomad != '')
```

Comment:
@ramprasadn Is there a difference between this statement vs. `if (!params.gnomad)`?

Reply:
It doesn't seem like it, but I do like your approach. Will change it in the next commit :)